src/ctirs/api/v1/stix_files/views.py

```diff
@@ -23,35 +24,26 @@ def get_api_get_stix_files_end(request):
     return get_text_field_value(request, 'end', default_value=None)
 
 # /api/v1/stix_files
-# GET/POSTの受付
 @csrf_exempt
+@api_key_auth
 def stix_files(request):
     try:
         if request.method == 'GET':
-            # stix_file一覧取得
             return get_stix_files(request)
         elif request.method == 'POST':
-            # stix_file追加
             return upload_stix_file(request)
         else:
             return HttpResponseNotAllowed(['GET', 'POST'])
     except Exception as e:
         return api_root.error(e)
 
 
-# 引数の日時文字列からdatetimeオブジェクトを作成
 def get_datetime_from_argument(s):
     return datetime.datetime.strptime(s, '%Y%m%d%H%M%S').replace(tzinfo=pytz.utc)
 
 
-# stix_file一覧取得
 # GET /api/v1/stix_files
 def get_stix_files(request):
```

Comment:
No decorator.

Reply:
`get_stix_files` calls from `stix_files` only.
`stix_files` has `@api_key_auth`.
So authentication is complete at this stage.